server/interaction/npc/calculated_npc.go

```diff
@@ -0,0 +1,30 @@
+package npc
+
+import (
+	"github.com/joshprzybyszewski/cribbage/logic/strategy"
+	"github.com/joshprzybyszewski/cribbage/model"
+	"github.com/joshprzybyszewski/cribbage/utils/rand"
+)
+
+var _ npcLogic = (*calcNPCLogic)(nil)
+
+type calcNPCLogic struct{}
+
+func (npc *calcNPCLogic) addToCrib(g model.Game, pID model.PlayerID, n int) []model.Card {
```

Comment:
Challenge for you: I think we can simplify this method to have the signature:

```go
func (npc *calcNPCLogic) addToCrib(hand []model.Card, isDealer bool) []model.Card
```

Reply:
Sure - I think I'll have this and `peg` return the action as well. Thoughts?